README.md

```diff
@@ -2,31 +2,132 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create a Gradle-based Java project in IntelliJ
+- Define the structure of a Gradle project
+- Define the parts of a build.gradle configuration file
 
 ## Introduction
+  
+Gradle is a fully open source build tool that emphasizes flexibility, performance,
+user experience, and dependency management.
+Gradle was designed for multi-project builds, which can grow to be large.
 
-A 1-2 sentence summary of what will be covered.
+Gradle configures a project using a Domain Specific Language (DSL) based on Groovy or Kotlin,
+which are Java-like languages that compile to Java Virtual Machine (JVM) bytecode.
 
-## Topic 1
+We will step through the process of using IntelliJ to create a Java project
+that will be built using Gradle. Gradle comes bundled with IntelliJ, so we don't have
+to install anything to get started.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Create a Gradle project
+  
+Open IntelliJ then select "File/New/Project" from the menu-bar
+or click the "New Project" button.
+    
+![new gradle project](https://curriculum-content.s3.amazonaws.com/6002/creating-a-gradle-project/newproject.png)
 
-## Topic 2
+1. Enter "example-app-2" for the project name.
+2. Enter a folder location to store the project.
+3. Select "Java" as the language.
+4. Select "Gradle" as the build system.
+5. Select a Java version that is at least Java 11.
+6. Select "Groovy" as the Gradle DSL.
+7. Check  "Add sample code"
+8. Enter "org.example" for the GroupId.
+9. Enter "example-app-2" for the ArtifactId.
+10. Press "Create".
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+IntelliJ creates the new project with the sample `Main` class.
+   
+![new project view](https://curriculum-content.s3.amazonaws.com/6002/creating-a-gradle-project/projectfolders.png)
+
+Press the green run button to execute the `main` method.
+The `.gradle` folder is created after the code compiles and runs.
+
+
+## Maven Directory Structure
+
+A Gradle project has a
+[standard directory structure](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html)
+for source code and configurations. 
+
+The following is a basic structure of a Gradle project:
+
+```text
+└── /example-app-2
+		├── src
+		│   ├── main
+		│   │   ├── java
+		│   │   └── resources
+		│   └── test
+		│       ├── java
+		│       └── resources
+		├── build.gradle
+		└── settings.gradle	
+```
+
+The structure of the `src` folder is similar to Maven's. Gradle does not use a `pom.xml` file
```

Comment:
Missing period after file.

"Gradle does not use a `pom.xml` file"

Reply:
fixed.